packages/worker/tests/integration/cron/verifyDeletedAccounts.test.ts

```diff
@@ -385,8 +386,8 @@ describe('[jobs - cron] verifyDeletedAccounts', () => {
                 ageRange6: '0',
                 male: '0',
                 female: '0',
-                undisclosed: '2',
-                totalGender: '2',
+                undisclosed: '3',
+                totalGender: '3',
```

Comment:
sorry, maybe I'm missing something here. Can you explain the change, please?

Reply:
with the latest changes, someone who hasn't a profile is considered `undisclosed`.
during this test, a user is deleted, and previously the test expected that the calculateCommunityDemographics ignore this deleted profile, but now, this profile should be calculated as `undisclosed`.